services/slack/intermediate.go

```diff
@@ -117,14 +117,28 @@ func (t *Transformer) TransformUsers(users []SlackUser) {
 
 	resultUsers := map[string]*IntermediateUser{}
 	for _, user := range users {
-		newUser := &IntermediateUser{
-			Id:        user.Id,
-			Username:  user.Username,
-			FirstName: user.Profile.FirstName,
-			LastName:  user.Profile.LastName,
-			Position:  user.Profile.Title,
-			Email:     user.Profile.Email,
-			Password:  model.NewId(),
+		var newUser *IntermediateUser
+
+		if user.IsBot {
```

Comment:
for bot what is user.id ?

Reply:
user.id was different from the bot ID. The messages created from bot were using bot ID

Reply:
check in case of bot what is the userID I think either it will be nil or equal to bot id .